### ISSUE TITLE
CP can't calculate path to start of the course

### ISSUE THREAD

**Issue by cosmin-margarit** [Courseplay/courseplay#5561]
Hello, I was working a contract job on field 4 (not owning field 4) planting potatoes, the course was set to return to the starting point when stopping, so inevitably the seeder ran out of seeds and tried tor return to the start of the course from where to start the refill course , when he ran out of seed he was about at the middle of the field (I've attached a photo to display where he stopped seeding) and from there he couldn't calculate the path to the start and got stuck in this error moving around on the field. He even got close to the start point but still couldn't find the path to it.
logs:
[cantCalculatePath.txt](https://github.com/Courseplay/courseplay/files/4732518/cantCalculatePath.txt)

I'll attach a video of the process to:
https://drive.google.com/file/d/1q9eC6SctIOX1mUkV5oqTcrhqm0SOKa9U/view?usp=sharing
![image](https://user-images.githubusercontent.com/32977353/83806094-9f4cea00-a6b9-11ea-9d33-b9985a007050.png)

**Comment by Tensuko** [comment]
> so inevitably the seeder ran out of seeds and tried tor return to the start of the course from where to start the refill course

CP Driver don't drive to start to refill. Do you have AutoDrive in use and also set that in adv settings ?
No Idea what exactly you did there...

**Comment by cosmin-margarit** [comment]
I used CP to start working the field near the AD point with Field 4. And I told CP to use AD for refill and  when it needs to refill to return to the start point where AD can take over and go tor refill and return, and then CP goes back where he left off. But at the moment when the seeder ran out of seeds, the CP driver didn't know to return to the start (as per the setting in CP hud). It did not trigger the Ad driver at any point because it did not reach the start.

**Comment by Tensuko** [comment]
> when it needs to refill to return to the start point where AD can take over and go tor refill and return

Where is the setting return to start for refill ?
The only setting I know for CP is return to start (wich means when work is done).
The other way CP won't know where he stopped.

**Comment by Tensuko** [comment]
Maybe we could change the tooltip to be more clear :)

**Comment by cosmin-margarit** [comment]
> Maybe we could change the tooltip to be more clear :)

Yeah, but every tutorial out there mentions that it's important to return to first point when refilling, and I understand that this is the case when using CP refill course merged into work course, but 
I also wanted this for AD, and I thought that if it returns to start when seeds run out without AD it would do the same with AD and based on that I had the AD route starting from the start point of the CP course,  so that after the driver returned he would know how to go to the refill station.

**Comment by Tensuko** [comment]
Oh wow wich tutorial say that ? Refill never had to return to start...
